lib/usb.js

```diff
@@ -16,6 +16,7 @@ var VENDOR_DEVICE_LIST = [
   {vid: 0x0a5c, pid: 0x21e8 }, // Broadcom BCM20702A0
```

Comment:
Should it be inserted close to this one ?

Reply:
I don't know weather the place is important, but it works for me

Reply:
please just relocate and then  eventually sort list alphabetically in other PR